integration_tests/suite/test_subscriptions.py

```diff
@@ -225,14 +223,19 @@ def test_when_create_invalid_subscription_then_400(self):
 
     def test_when_create_http_subscription_then_subscription_no_error(self):
         webhookd = self.make_webhookd(VALID_TOKEN)
+        tenant_uuid = "ffffffff-ffff-ffff-ffff-ffffffffffff"
```

Comment:
The tenant used to create the subscription is extracted from the token. So we should import information from the token used. But this token is hardcoded in wazo-auth-mock, se we use constant, and in this case we should import WAZO_TENANT from the base helper, not an arbitrary specific constant for this test.

Reply:
done